mysql-test/main/mysql_tzinfo_to_sql_symlink.test

```diff
@@ -63,7 +64,7 @@ SELECT COUNT(*) FROM time_zone_transition;
 SELECT COUNT(*) FROM time_zone_transition_type;
 SELECT COUNT(*) FROM time_zone_leap_second;
 
-if ($is_embedded) {
+if ($replace_wsrep_cols) {
 --replace_column 1 0 2 0
 }
 SELECT @wsrep_is_on,  @wsrep_cannot_replicate_tz, @save_wsrep_on, @save_sql_log_bin, @@SQL_LOG_BIN;
```

Comment:
@grooverdan added these repeated queries to verify the state of various variables in

https://github.com/MariaDB/server/commit/13e77930e615f05cc74d408110e887b00e1abcc9#diff-7f0bc5adad1095667313981a760f8a268e13a1ba78d6d5bddf8b9deb5bec1ad2R64-R68

I don't fully understand why it's particularly useful or necessary to repeatedly verify the state of these variables at different points along the test.

If others agree, an alternative solution here would be to simply remove some or all of these repeated queries and their results. This would likely make it easier for others to read, maintain, and update in the future, without any apparent reduction in the value of the test for verifying the TZ tables.